### PR TITLE
[ES] Add `B` missing stubs (4)

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

Missing B stubs from the `Missing stubs (up-to-date in English)` section according to the osu! wiki status in spanish.

32 stubs in total, divided into 4 pull requests of 8 stubs each.

### CODE REVIEW

File: wiki/Beatmapping/Mapping_techniques/Blanket/es.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - blanketing
+---
+
+# Blanket
+
+::: Infobox
+![](img/blanket.png "Un blanket del objeto 2 usando el sliderbody del objeto 1")
+:::
+
+Un **blanket** es el efecto creado cuando un [sliderbody](/wiki/Gameplay/Hit_object/Slider/Sliderbody) envuelve la forma de otro [objeto](/wiki/Gameplay/Hit_object). El blanketing se puede realizar de muchas maneras para adaptarse al estilo visual de un [beatmap](/wiki/Beatmap). Puedes encontrar una guía para hacerlos en el [artículo sobre las técnicas de mapping](/wiki/Beatmapping/Mapping_techniques/Making_good_sliders#beat-blankets).
```
Comment (agatemosu) [inline wiki/Beatmapping/Mapping_techniques/Blanket/es.md:14]:
```suggestion
Un **blanket** es el efecto creado cuando un [sliderbody](/wiki/Gameplay/Hit_object/Slider/Sliderbody) envuelve la forma de otro [objeto](/wiki/Gameplay/Hit_object). El blanketing se puede realizar de muchas maneras para adaptarse al estilo visual de un [beatmap](/wiki/Beatmap). Puedes encontrar una guía para hacerlos en el [artículo de técnicas de mapping](/wiki/Beatmapping/Mapping_techniques/Making_good_sliders#beat-blankets).
```

File: wiki/Beatmapping/Mapping_techniques/Overlap/es.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
```
Comment (agatemosu) [inline wiki/Beatmapping/Mapping_techniques/Overlap/es.md:3]:
```suggestion
tags:
  - superponer
---
```

File: wiki/Bit_rate/es.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+---
+
+# Tasa de bits
+
+*Para conocer las regulaciones relativas a la tasa de bits, véase: [Criterios de clasificación](/wiki/Ranking_criteria)*\
+*Para obtener ayuda para volver a codificar un audio a una tasa de bits diferente, véase: [Guía de edición de audio](/wiki/Guides/Audio_editing).*
+
+La **tasa de bits** es una medida del rendimiento de datos de un archivo multimedia en unidades de bits a lo largo del tiempo. En osu!, esto es relevante cuando se habla de la calidad de los archivos de audio: el audio de mayor calidad normalmente requiere una tasa de bits más alta, mientras que el audio de menor calidad puede usar una tasa de bits más baja.
+
+Las tasas de bits de los archivos de audio suelen oscilar entre 96 kbps (kilobits por segundo) y 320 kbps. Los [criterios de clasificación](/wiki/Ranking_criteria) requieren que el audio de la canción esté en el rango de 128 a 192 kbps, y [AiMod](/wiki/Client/Beatmap_editor/AiMod) advertirá si este no es el caso.
```
Comment (agatemosu) [inline wiki/Bit_rate/es.md:12]:
```suggestion
La tasa de bits de los archivos de audio suele oscilar entre 96 kbps (kilobits por segundo) y 320 kbps. Los [criterios de clasificación](/wiki/Ranking_criteria) requieren que el audio de la canción esté en el rango de 128 a 192 kbps, y [AiMod](/wiki/Client/Beatmap_editor/AiMod) advertirá si este no es el caso.
```

File: wiki/Beatmapping/Slider_tick_rate/es.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+stub: true
+tags:
+  - ticks
+  - marcas
+---
+
+# Tasa de la marca del slider
+
+La *tasa de la marca del slider* es la velocidad a la que aparecen las [marcas de los sliders](/wiki/Gameplay/Hit_object/Slider/Slider_tick) en los [sliders](/wiki/Gameplay/Hit_object/Slider). Por ejemplo, la velocidad de la tasa de las marcas del slider en 1 significa que las marcas del slider aparecen una vez por pulso. Sin embargo, como las marcas de los sliders no aparecen en [sliderheads](/wiki/Gameplay/Hit_object/Slider/Sliderhead) y [sliderends](/wiki/Gameplay/Hit_object/Slider/Slidertail), algunos sliders no contienen ninguna marca de slider. Por ejemplo, un slider de 1 pulso no contiene ninguna marca de slider con la tasa de la marca del slider establecida en 1. Las tasas típicas de las marcas del slider incluyen 1, 2 y 4. El 3 también está disponible pero rara vez se usa.
```
Comment (agatemosu) [inline wiki/Beatmapping/Slider_tick_rate/es.md:10]: reads better for me
```suggestion
La *frecuencia de marcas del slider* es la tasa a la que aparecen las [marcas del slider](/wiki/Gameplay/Hit_object/Slider/Slider_tick) en los [sliders](/wiki/Gameplay/Hit_object/Slider). Por ejemplo, la velocidad 1 significa que las marcas aparecen una vez por pulso. Sin embargo, como las marcas del slider no aparecen en los [sliderheads](/wiki/Gameplay/Hit_object/Slider/Sliderhead) ni en los [sliderends](/wiki/Gameplay/Hit_object/Slider/Slidertail), algunos sliders no contienen ninguna marca. Por ejemplo, un slider de 1 pulso no contendrá ninguna marca con la frecuencia establecida en 1. Las frecuencias de marca de slider típicas incluyen 1, 2 y 4. 3 también está disponible, pero rara vez se usa.
```

File: wiki/Beatmapping/Slider_tick_rate/es.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+stub: true
+tags:
+  - ticks
+  - marcas
+---
+
+# Tasa de la marca del slider
```
Comment (agatemosu) [inline wiki/Beatmapping/Slider_tick_rate/es.md:8]:
```suggestion
# Tasa de marcas del slider
```

File: wiki/Beatmapping/Slider_tick_rate/es.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+
+La *frecuencia de marcas del slider* es la tasa a la que aparecen las [marcas del slider](/wiki/Gameplay/Hit_object/Slider/Slider_tick) en los [sliders](/wiki/Gameplay/Hit_object/Slider). Por ejemplo, la velocidad 1 significa que las marcas aparecen una vez por pulso. Sin embargo, como las marcas del slider no aparecen en los [sliderheads](/wiki/Gameplay/Hit_object/Slider/Sliderhead) ni en los [sliderends](/wiki/Gameplay/Hit_object/Slider/Slidertail), algunos sliders no contienen ninguna marca. Por ejemplo, un slider de 1 pulso no contendrá ninguna marca con la frecuencia establecida en 1. Las frecuencias de marca de slider típicas incluyen 1, 2 y 4. 3 también está disponible, pero rara vez se usa.
```
Comment (agatemosu) [inline wiki/Beatmapping/Slider_tick_rate/es.md:10]:
```suggestion
La *tasa de marcas del slider* es la frecuencia a la que aparecen las [marcas del slider](/wiki/Gameplay/Hit_object/Slider/Slider_tick) en los [sliders](/wiki/Gameplay/Hit_object/Slider). Por ejemplo, una tasa de 1 significa que las marcas aparecen una vez por pulso. Sin embargo, como las marcas del slider no aparecen en los [sliderheads](/wiki/Gameplay/Hit_object/Slider/Sliderhead) ni en los [sliderends](/wiki/Gameplay/Hit_object/Slider/Slidertail), algunos sliders no contienen ninguna marca. Por ejemplo, un slider de 1 pulso no contendrá ninguna marca con la tasa establecida en 1. Las tasas de marcas del slider típicas incluyen 1, 2 y 4. 3 también está disponible, pero rara vez se usa.
```

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**